nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/resources/OSGI-INF/search-core-types-contrib.xml

```diff
@@ -10,17 +10,9 @@
 
   <extension target="org.nuxeo.ecm.core.schema.TypeService" point="doctype">
 
-    <facet name="SavedSearch" perDocumentQuery="false">
-      <schema name="common" />
-      <schema name="dublincore" />
-      <schema name="uid" />
-    </facet>
-
     <!-- For default search -->
-    <doctype name="DefaultSearch" extends="Document">
-      <facet name="SavedSearch" />
+    <doctype name="DefaultSearch" extends="SavedSearch">
```

Comment:
You should probably `<require>org.nuxeo.ecm.platform.search.CoreExtensions</require>` so that `SavedSearch` is defined first, although it may work without it, not sure.